packages/react/src/patterns/blocks/LeadSpaceBlock/__tests__/LeadSpaceBlock.test.js

```diff
@@ -0,0 +1,85 @@
+import LeadSpaceBlock from '../LeadSpaceBlock';
```

Comment:
This might not have any effect on coverage amount if we're already using storyshots, especially if you're only doing a shallow render.

Reply:
In this case, we don't need this test?

Reply:
Can you try removing and check if the coverage amount has changed? My suspicion is that it isn't necessary and can be deleted.

Reply:
just testing the rendering, deleted the test.